CDC-Data-Reconciliation-Frontend/src/components/Report.jsx

```diff
@@ -1,11 +1,168 @@
-import { useState, useEffect } from "react"
+import { useState, useEffect, useMemo } from "react"
+import DebouncedInput from "./DebouncedInput"
+import Filter from "./Filter"
 import config from "../config.json"
+import { FaSortDown, FaSortUp } from "react-icons/fa6"
```

Comment:
I was going to suggest using Material UI, but this is actually a better design choice!

Reply:
Yeah I also use material UI icons below, MD. I just couldn't find a good sorting up and down icon from Material design icons